pybricksdev/ble.py

```diff
@@ -29,14 +29,15 @@ async def find_device(name: str, timeout: float = 5) -> BLEDevice:
         asyncio.TimeoutError:
             Device was not found within the timeout.
     """
-    print("Searching for {0}".format(name))
+    print("Searching for {0}".format(device_or_address))
 
     queue = asyncio.Queue()
 
     def set_device_discovered(device: BLEDevice, _: AdvertisementData):
-        if device.name != name:
+        if device_or_address in [device.name, device.address]:
+            queue.put_nowait(device)
+        else:
```

Comment:
`else: return` is not needed since there is nothing in the function after this.

Reply:
that is ok